models/aeif_cond_alpha.h

```diff
@@ -378,7 +365,57 @@ aeif_cond_alpha::set_status( const DictionaryDatum& d )
   S_ = stmp;
 }
 
+/**
+ * Function computing right-hand side of ODE for the ODE solver.
+ * @param y State vector (input).
+ * @param f Derivatives (output).
+ */
+inline void
+aeif_cond_alpha::aeif_cond_alpha_RK5_dynamics( const double y[], double f[] )
```

Comment:
I'd suggest to move this method (back) to the *.cpp file. Since it is only ever called from code within the cpp file, it can be inlined entirely within that file, no need to have it in the h-file.

Another question is whether inlining helps performance. The method is quite large, so function call overhead is probably limited compared to the total runtime per call, while code-bloat due to inlining is probably considerable, given that the method is called seven times in the inner update loop, so inlining will paste close to 300 lines of code into the loop.  Would be interesting to test this empirically.